src/browser/modules/D3Visualization/components/styled.tsx

```diff
@@ -181,12 +181,13 @@ export const StyledStatusBarWrapper = styled.div`
   display: none;
 `
 
-export const StyledStatusBar = styled.div`
+export const StyledStatusBar = styled.div<{ fullscreen?: boolean }>`
   min-height: 39px;
   line-height: 39px;
   color: ${props => props.theme.secondaryText};
   font-size: 13px;
-  position: absolute;
+  position: ${props => (props.fullscreen ? 'fixed' : 'absolute')};
+  z-index: ${props => (props.fullscreen ? 1 : 0)};
```

Comment:
the default for z-index is `auto`, so we should use `unset` or `auto` instead of `0` here (and elsewhere when the intention is to have it perform as it was before)